src/decorators/authenticate-user.decorator.ts

```diff
@@ -1,12 +1,14 @@
 import {
+  BindingKey,
   Constructor,
   MetadataInspector,
   MethodDecoratorFactory,
 } from '@loopback/context';
-import {Request} from '@loopback/rest';
+import { Request } from '@loopback/rest';
 
-import {USER_AUTHENTICATION_METADATA_KEY} from '../keys';
-import {AuthenticationMetadata} from '../types';
+import { USER_AUTHENTICATION_METADATA_KEY } from '../keys';
```

Comment:
Linting should remove leading and trailing spaces between braces

Reply:
Done